connect-spring-boot-app/src/main/resources/application.yml

```diff
@@ -4,13 +4,22 @@ spring:
     producer:
       key-serializer: org.apache.kafka.common.serialization.StringSerializer
       value-serializer: io.confluent.kafka.serializers.KafkaAvroSerializer
+    consumer:
+      group-id: spring-boot-consumer-group
```

Comment:
is this needed?

Reply:
Removed default consumer group from application.yml to force consumers to specify their own